R/ResultsDataModel.R

```diff
@@ -21,12 +21,14 @@
 #' @param packageVersion The version number of cohort diagnostics
 #' @param modelVersion   The version of the results data model
 #' @param packageName    The name of the R package whose output model we are documenting.
+#' @param schemaBindingVariable     The name of the schema binding parameter
 #' 
 #' @export
 createDdl <- function(packageName,
                       packageVersion,
                       modelVersion,
-                      specification){
+                      specification,
+                      schemaBindingParam = "databaseSchema"){
```

Comment:
should be `schemaBindingVariable = "databaseSchema"` based on documenation?

Reply:
We could call it is as 'databaseSchema' to represent a generic database + schema combination

Reply:
> should be `schemaBindingVariable = "databaseSchema"` based on documenation?

good catch @gowthamrao, can you please review again?